services/order.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: datetime = None) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user, created_at=date if date else datetime.now())
+        tickets_instances = []
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(id=ticket["movie_session"])
```

Comment:
The `create_order` function uses `MovieSession.objects.get(id=ticket['movie_session'])`, which will raise a `DoesNotExist` exception if no movie session with the given ID is found. Consider handling this exception to improve robustness.